prowler/providers/aws/services/eventbridge/eventbridge_bus_cross_account_access/eventbridge_bus_cross_account_access.metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+  "Provider": "aws",
+  "CheckID": "eventbridge_bus_cross_account_access",
+  "CheckTitle": "Ensure that AWS EventBridge event buses do not allow unknown cross-account access for delivery of events.",
+  "CheckType": [],
+  "ServiceName": "eventbridge",
+  "SubServiceName": "eventbus",
```

Comment:
I think @pedrooot put this in upper case, please check it also for the other checks.

Reply:
Maybe we need to remove the `SubServiceName` as you mentioned.